### PR TITLE
Fix Bug #53 - Personal folders list spaces

### PR DESCRIPTION
**Pull request by vasylmrnchk**

### CODE REVIEW

File: frontend/src/app/modules/workspace/personal/personal.component.scss [hunk 1]
```diff
@@ -23,10 +23,6 @@ mat-list-item:hover {
   border: 2px solid #fb6d3a;
 }
 
-mat-list-item:nth-child(3) {
```
Comment (Jyuart) [inline frontend/src/app/modules/workspace/personal/personal.component.scss:26]: Nice, but it creates this alignment issue
Can you try to deal with it?
image

Comment (vasylmrnchk) [reply frontend/src/app/modules/workspace/personal/personal.component.scss:26]: oh, of course. i fix it

Comment (vasylmrnchk) [reply frontend/src/app/modules/workspace/personal/personal.component.scss:26]: In my screen in the row only two element, and why i did not see the difference. I wiil try solve it

**Review by Jyuart — APPROVED** [review]
Very nice, thank you!

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**